src/winetricks

```diff
@@ -5037,7 +5102,8 @@ winetricks_init()
 
     winetricks_get_sha256sum_prog
 
-    winetricks_get_platform
+    # winetricks_get_platform got updated and in addition to W_PLATFORM export it outputs the platform which seems unexpected here
```

Comment:
So remove the printf's you added..

Reply:
i believe that calling the function directly is more sane assuming situation in which W_PLATFORM is not exported.